ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource

```diff
@@ -343,3 +382,267 @@ Set RHODS Users Group To Inexistent Group
     ...    to the given inexistent group
     Set Access Groups Settings    admins_group=${STANDARD_ADMINS_GROUP}
     ...    users_group=${CUSTOM_INEXISTENT_GROUP}
+
+Click On Create Accelerator Profile Button
+    [Documentation]    Click on the "Add new accelerator profile" (when there is none) or
+    ...                the "Create accelerator profile" (when there is at least 1
+    ...                previously created).
+    ...                view: Settings -> Accelerator profiles
+    ${empty_list} =  Run Keyword And Return Status    Wait Until Page Contains Element
+    ...    ${CREATE_ACCELERATOR_PROFILE_EMPTY_LIST_BTN}  timeout=5s
+    IF  ${empty_list}==True
+        Click Button   ${CREATE_ACCELERATOR_PROFILE_EMPTY_LIST_BTN}
+    ELSE
+        Click Button   ${CREATE_ACCELERATOR_PROFILE_BTN}
+    END
+
+Click On Edit Accelerator Profile
+    [Documentation]    Click on the the 3 dots button and then on the "Edit" button of an specific Accelerator Profile
+    ...                from the grid
+    ...                view: Settings -> Accelerator profiles
+    [Arguments]   ${name}
+    Search for accelerator profile in grid    ${name}
+    Wait Until Page Contains Element    ${ACCELERATOR_PROFILE_GRID_COLUMN_NAME}//*[text()='${name}']/ancestor::tr//button[@aria-label='Kebab toggle']
+    Click Button    ${ACCELERATOR_PROFILE_GRID_COLUMN_NAME}//*[text()='${name}']/ancestor::tr//button[@aria-label='Kebab toggle']
+    Wait Until Page Contains Element    ${EDIT_ACCELERATOR_PROFILE_BTN}
+    Click Button    ${EDIT_ACCELERATOR_PROFILE_BTN}
```

Comment:
have you tried re-using `ODHDashboard.Click Action From Actions Menu` keyword? If not, could you pls do it in order to spare new redudant code lines?